### DISCUSSION TITLE
Zsh looks strange over ssh in kitty

### DISCUSSION THREAD

**Discussion by Demianeen** [kovidgoyal/kitty#7063] [Q&A] [answered]
I have noticed that the Zsh over ssh works very strangely in kitty, but not in iterm:
https://github.com/kovidgoyal/kitty/assets/51330172/298d554e-bbc7-430d-8162-ebfa6fc38b4d

My kitty conf is pretty small and consists only from colors and fonts settings, so it should be because of it:
```
:     font_family      Hark Nerd Font Mono
:     bold_font        auto
:     italic_font      auto
:     bold_italic_font auto
# font_size 11.0


# Color Scheme {{{
foreground #c0caf5
background #222435
selection_foreground #c0caf5
selection_background #33467c
url_color #73daca
bold #eeeeee

# ask for confirmation before close
confirm_os_window_close 0

# Cursor
cursor #c0caf5
cursor_text_color #15161e

#: The color table {{{
# there are normal and brighter version of the color

#: black
color0 #000000
color8 #414868

#: red
color1 #f7768e
color9 #f7768e

#: green
color2  #9ece6a
color10 #9ece6a

#: yellow
color3  #e0af68
color11 #e0af68

#: blue
color4  #7aa2f7
color12 #7aa2f7

#: magenta
color5  #bb9af7
color13 #bb9af7

#: cyan
color6  #7dcfff
color14 #7dcfff

#: white
color7  #a9b1d6
color15 #c0caf5
```

Why can it work that way? Are there any issues with kitty over ssh?

**Comment by kovidgoyal** [comment, marked as answer]
https://sw.kovidgoyal.net/kitty/faq/#i-get-errors-about-the-terminal-being-unknown-or-opening-the-terminal-failing-or-functional-keys-like-arrow-keys-don-t-work

**Reply by Demianeen** [reply]
But why is that work that way? The docs say that the kitty doesn't have terminfo, but why then other terminals work over ssh?
